app/helpers/shift_commerce/bundle_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ShiftCommerce
+  module BundleHelper
+    BUNDLE_CACHE_VERSION = "v1"
+
+    def bundle_cache_version
```

Comment:
This method now can be overridden by the repos who import this gem and update their version.